src/pages/Login/test_utils.ts

```diff
@@ -0,0 +1,19 @@
+import { act } from "@testing-library/react";
+import userEvent, { TargetElement } from "@testing-library/user-event";
+
+function fillInput(container: HTMLElement, phoneInput: string) {
+  const input = container.querySelector("ion-item > ion-input");
+  act(() => {
+    input!.dispatchEvent(
+      new CustomEvent("ionChange", {
+        detail: { value: phoneInput },
+      })
+    );
+  });
+}
```

Comment:
This makes the assumption that the container has `ion-item > ion-input` inside it, so maybe rename the method `fillIonInput` to be clearer, and you can probably do `querySelector('ion-input')` that way `ion-item` won't be required